tests/test_cli.py

```diff
@@ -576,7 +576,9 @@ def test_cli_fetch_command_with_es_backend(es):
 
     runner = CliRunner()
     es_hosts = ",".join(ES_TEST_HOSTS)
-    command = f"-v ERROR fetch -b es --es-hosts {es_hosts} --es-index {ES_TEST_INDEX}"
+    es_client_options = "verify_certs=True"
```

Comment:
Maybe also test the CLI when providing multiple client options?

Reply:
That's true. I added a new test with multiple client options.